scripts/install-tools.sh

```diff
@@ -78,7 +78,7 @@ install_golangci() {
       install=true
   fi
   if [[ "$install" == true ]]; then
-      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/cc3567e3127d8530afb69be1b7bd20ba9ebcc7c1/install.sh \
+      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/refs/tags/v2.2.2/install.sh \
```

Comment:
Tags on GitHub aren’t immutable, using the sha is a more secure approach and it would be good to maintain the same approach.

Reply:
Thanks @Raffo 
Got it